tools/installer/common/installer.include

```diff
@@ -142,9 +142,9 @@ stage_install_common() {
   fi
 
   # l10n paks
-  cp -a "${BUILDDIR}/locales" "${STAGEDIR}/${INSTALLDIR}/"
-  find "${STAGEDIR}/${INSTALLDIR}/locales" -type f -exec chmod 644 '{}' \;
-  find "${STAGEDIR}/${INSTALLDIR}/locales" -type d -exec chmod 755 '{}' \;
+  cp -a "${BUILDDIR}/locales/xwalk" "${STAGEDIR}/${INSTALLDIR}/"
```

Comment:
You're likely creating a directory called `${STAGEDIR}/${INSTALLDIR}/xwalk` now, not `${STAGEDIR}/${INSTALLDIR}/locales/xwalk`.